test/app.test.ts

```diff
@@ -115,5 +116,9 @@ test('simple calculate', async () => {
     .send({ measure: mockMeasureBundle, patients: mockPatientBundles })
     .expect(200);
 
-  expect(response.body).toEqual(mockResult.results);
+  expect(response.body.results).toEqual(mockResult.results);
 });
+function parseBundle(filePath: string): R4.IBundle {
```

Comment:
Is the `parseBundle()` function needed in this test file? I'm seeing a warning that it's defined but never used

Reply:
Good point. I may end up needing it for the second half of this task to test the error/warning reporting, but makes sense to remove it for now.